hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/DFSConfigKeys.java

```diff
@@ -136,6 +136,11 @@ public class DFSConfigKeys extends CommonConfigurationKeys {
       "dfs.datanode.ec.reconstruct.write.bandwidthPerSec";
   public static final long DFS_DATANODE_EC_RECONSTRUCT_WRITE_BANDWIDTHPERSEC_DEFAULT =
       0; // A value of zero indicates no limit
+  public static final String DFS_DATANODE_COPY_BLOCK_CROSS_NAMESPACE_SOCKET_TIMEOUT_MS_KEY =
```

Comment:
Is this one config item same as CopyOp time out and is it necessary?

Reply:
This is used  for copyBlockCrossNamespaceExecutor task timeout.    And it  not ,  a task will block until it sucess or failed.  This can be removed or retain.